src/services/SearchCarousel/SearchCarouselServices.types.ts

```diff
@@ -0,0 +1,16 @@
+type searchCarouselName = {
```

Comment:
This is generic for the entire app. Maybe move i18n folder

Reply:
Moved to i18n folder